### DISCUSSION TITLE
모바일 제페토 에러

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#327] [5. Q&A]
![스크린샷(309)](https://user-images.githubusercontent.com/68143858/152666583-a352d5a4-501b-4a26-bfb6-14f9836ce0c8.png)
![스크린샷(310)](https://user-images.githubusercontent.com/68143858/152666585-07290297-f8bb-429e-91b0-438ce94b1e70.png)
![KakaoTalk_20220206_115246238](https://user-images.githubusercontent.com/68143858/152666587-d5a4b5a8-885c-4d19-9057-37e19082eb43.jpg)

multiplay room 구현 중입니다.

유니티에서는 잘 작동하는 거 같은데, 동시 접속 확인하려고 모바일로 접속하니까 계속 파란 화면만 뜨고 팅깁니다. 

왜 이런 에러가 뜨는 것일까요..? (같은 네트워크 사용했습니다!)

**Comment by minjujuu** [comment]
저는 그럴 때 방화벽 해제해서 해결했었어요! 한 번 시도해보시면 좋을 거 같아요
![image](https://user-images.githubusercontent.com/79794414/152668019-bdc86dbb-32c2-4014-b673-8a9b7c54b578.png)
![image](https://user-images.githubusercontent.com/79794414/152668084-d432e7ac-d846-4c73-8179-927363637d89.png)

**Reply by hipsc159** [reply]
감사합니다! 들어가는게 해결됐네요!!
그런데 들어가고 나니까 캐릭터가 안뜨고 그냥 화면이 정지되어 있네요.
죄송하지만 혹시 이 부분에 대해서 아시는게 있을까요..???
![KakaoTalk_20220206_143806984](https://user-images.githubusercontent.com/68143858/152669709-86c7bc62-2558-4935-93fe-c68bfd062f77.jpg)

**Reply by minjujuu** [reply]
![image](https://user-images.githubusercontent.com/79794414/152669836-271c0f96-1273-4c42-a8c6-776af5d4b4ac.png)

혹시 서버 켜져있을까요?
저도 가끔 깜박해서 서버 끄고 들어가면 화면 정지된 상태로 나왔었어요

**Reply by hipsc159** [reply]
와 진짜 감사합니다!! 해결됐습니다!
